ansible/library/get_feature.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/python
+
+DOCUMENTATION = '''
+module:         get_feature
+version_added:  "1.0"
```

Comment:
Most likely this would be interpreted by people as "the module added in ansible version 1.0". It doesn't make sense. Suggest to remove this line.

Reply:
done